optical/converter/yolo.py

```diff
@@ -8,15 +8,18 @@
 import os
 import warnings
 from pathlib import Path
-from typing import Union
+from typing import Dict, Union
 
 import imagesize
-import yaml
 import numpy as np
 import pandas as pd
+import yaml
+from tqdm.auto import tqdm
+from joblib import Parallel, delayed
+from functools import partial
```

Comment:
imports are not sorted in appropriate order, please use `isort` for the same. You can configure isort in vscode as well.

Reply:
sorted the imports. Thanks.